packages/oui-form-actions/README.md

```diff
@@ -0,0 +1,48 @@
+# Form Actions
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+```html
+<oui-form-actions
+  on-submit="$ctrl.submit()"
+  on-cancel="$ctrl.cancel()"
+></oui-form-actions>
+```
+
+## API
+
+| Attribute     | Type     | Binding | One-time Binding | Values                 | Default   | Description                      |
+| ----          | ----     | ----    | ----             | ----                   | ----      | ----                             |
+| on-submit     | function | &       |                  |                        |           | submit form action               |
+| on-cancel     | function | &       |                  |                        |           | cancel form action               |
```

Comment:
Can this button be optional in the guidelines?

Reply:
In CX guidelines nothing mention it. So I considered it as not optional. (which makes sense IMO.)